src/ui/layout/tests/Layout.test.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { render, screen, fireEvent } from '@testing-library/react';
+
+import { Layout } from "..";
+
+
+describe('<Layout>', () => {
+    it("shows children", () => {
+        const child = <div data-testid="child">Test</div>;
+        render(<Layout>{child}</Layout>)
+        expect(screen.getByTestId("child")).toBeDefined();
+    })
+
+    it("style pass correctly", () => {
+        const child = <div data-testid="child">Test</div>;
+        const { container } = render(<Layout className={"foo"}>{child}</Layout>)
+        expect(container.firstChild).toHaveClass('foo');
+    })
+
+    it("scroll var change height property", () => {
+        const child = <div data-testid="child">Test</div>;
```

Comment:
If test is more complicated - add enters between ARANGE, ACT, ASSERT steps.

Also test should be named - displays browser scroll for overgrown property

Reply:
done